internal/controller/assets/deployment.go

```diff
@@ -858,3 +884,10 @@ func getNodeAffinity(nodeAffinity *corev1.NodeAffinity) *corev1.Affinity {
 		},
 	}
 }
+
+func getIARAgentServicePort(falconImageAnalyzer *falconv1alpha1.FalconImageAnalyzer) int32 {
+	if falconImageAnalyzer.Spec.ImageAnalyzerConfig.IARAgentService.HTTPPort != 0 {
+		return falconImageAnalyzer.Spec.ImageAnalyzerConfig.IARAgentService.HTTPPort
+	}
+	return 8001 // Default port
```

Comment:
Can we define this default once in a central location? Maybe in pkg/common/constants.go?